src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/rowconfig.html

```diff
@@ -8,7 +8,12 @@
       <div class="alert alert-warn ng-scope" ng-show="nameChanged">
           Modifying a row configuration name will result in loss of
           data for any existing content that is based on this configuration.
+          Modifying only the label will not result in data loss.
```

Comment:
After quick ping on Slack @clausjensen modified the copy to be wrapped in P's & the second sentence to be in bold for now as think it needs to be easier to see that modifying this new label field is OK to do so.